src/collapse/perf.rs

```diff
@@ -231,24 +464,20 @@ impl Folder {
                 if event.ends_with(':') {
                     let event = &event[..(event.len() - 1)];
 
-                    if let Some(ref filter) = self.opt.event_filter {
-                        if event != filter {
-                            if let EventFilterState::Defaulted = self.event_filtering {
-                                // only print this warning if necessary:
-                                // when we defaulted and there was
-                                // multiple event types.
-                                warn!("Filtering for events of type: {}", event);
-                                self.event_filtering = EventFilterState::Warned;
+                    match self.event_filter {
```

Comment:
Why change this from an `if let Some else` to a `match`?

Reply:
Eh, I just prefer `match` to `if let Some(foo) = bar {} else {}`. Would you like me to use the `if let Some...` instead?

Reply:
It's fine either way. I was just surprised to see it actively changed from one to the other as it increases the size of the diff. I'm fine with you using whatever you prefer for _new_ code, but I think my preference would be to not change what's used in existing code if it's not necessary.

Reply:
Oh, my bad. When I responded before I didn't realize that the **only** change I made here was the cosmetic turning `if let Some(..` into `match`. If that's the case, than, yah, I shouldn't make that kind of change without something substantive changing as well.